README.md

```diff
@@ -211,7 +212,7 @@ The software will recognize all previous labeling decisions, and a new model can
 |      [FASTREAD](#fastread)      |                TF-IDF                |               SVM                |          S, A, W, M          |      C, U      |
 |        [Rayyan](#rayyan)        |     :grey_question:<sup>4</sup>      |               SVM                | :grey_question:<sup>4</sup>  |      C, U      |
 |  [RobotAnalyst](#robotanalyst)  |        TF-IDF + BOW + LDA2vec        |               SVM                | :grey_question:<sup>5</sup>  |  R, C, U, Cl   |
-|  [SWIFT-Review](#swift-review)  |             TF-IDF + LDA             |                LL                | S:grey_question:<sup>6</sup> |    C, U, Cl    |  
+|  [SWIFT-Active Screener](#swift-activescreener)  |             TF-IDF + LDA             |                LL                | S:grey_question:<sup>6</sup> |    C, U, Cl    |  
```

Comment:
- Feature extractor: TF-IDF (no LDA)
- In the paper they mention that query strategy is only certainty based

Reply:
solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53